### ISSUE TITLE
BlenderBIM : Can't move when I open IFC files

### ISSUE THREAD

**Issue by SouIchimiya** [IfcOpenShell/IfcOpenShell#2915]
Hi, I'm Japanese student and studying BlenderBIM.

When I try to open an IFC file, Blender stops working.
The file itself is a building model I created, I created a representation for a plan view, and when I assigned it, it suddenly became too heavy to open.
I personally think that the representation I created is causing the problem.
I also want to export the plan view, but eventually that too stops working and I can't output it. Please help.

I wrote this pattern in HTML (or CSS).
```
<pattern id="concrete1" width="15" height="1" patternTransform="rotate(45 0 0)" patternUnits="userSpaceOnUse">
    <path style="fill: white;" d="M 0 0 3 0 3 3 0 3" />
    <line x1="1" y1="0" x2="1" y2="3" style="stroke:black; stroke-width:0.25" />
    <line x1="1.5" y1="0" x2="1.5" y2="3" style="stroke:black; stroke-width:0.25" />
    <line x1="2" y1="0" x2="2" y2="3" style="stroke:black; stroke-width:0.25" />
```
Blender ver. : 3.4
BlenderBIM ver. : 0.0.230304

OS : Windows
CPU : intel core i5 13600k
GPU : NVIDIA Geforece GTX 1650s
Memory : 16GB

**Comment by SouIchimiya** [comment]
This is an IFC file that will not open.
https://www.dropbox.com/s/iw6isfxgxc9gzei/BSITest.ifc?dl=0

**Comment by Moult** [comment]
@SouIchimiya if you enable advanced mode then disable multiprocessing, it loads fine.

@aothms will know exactly why multiprocessing fails though.

How did you create the model? There are quite a number of validation issues.

**Comment by SouIchimiya** [comment]
@Moult 
Thanks to your advice I was able to open it!
However, some walls have disappeared.
![image](https://user-images.githubusercontent.com/129352032/228877915-7da4fcac-419c-45cb-a768-587c0d357990.png)

The model was created based on a YouTube video by IFC architect.
The walls and fittings were created from IFC Element Type, and I did not know how to create stairs, slabs, columns, and beams, so I assigned an IFC Class to the Blender model and converted it to a BIM object.
